src/main/java/mousio/etcd4j/responses/EtcdHealthResponse.java

```diff
@@ -0,0 +1,26 @@
+package mousio.etcd4j.responses;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Etcd Health response
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class EtcdHealthResponse implements EtcdResponse {
+
+	// The json
+	public static final EtcdResponseDecoder<EtcdHealthResponse> DECODER =
+		EtcdResponseDecoders.json(EtcdHealthResponse.class);
+
+	private final String health;
+
+	EtcdHealthResponse(
+		@JsonProperty("health") String health) {
```

Comment:
Yes it is a string but if the values are true/false it may be better to use a boolean as ``` resp.isHealth() ``` looks much nicer that ``` "true".equals(resp.getHealth()) ```